Core/Utilities/Profiler.py

```diff
@@ -0,0 +1,200 @@
+"""
+Profiling class for updated information on process status
+"""
+
+__RCSID__ = "$Id$"
+
+import psutil
```

Comment:
`psutil` is NOT in the externals: please make a PR adding it to https://github.com/DIRACGrid/Externals/blob/master/ServerPackages/dirac-make.py

And in this same PR you should add psutil to requirements.txt